.github/workflows/benchmarks.yml

```diff
@@ -3,6 +3,7 @@ name: Benchmarks
 on:
   push:
     branches: 'master'
+    paths: '!docs/**'
```

Comment:
Now all CI is broken… @fpoli @Aurel300

Reply:
I'm on it.

Reply:
The correct filter might be `paths-ignore: 'docs/**'; I'll try.

Reply:
@vakaras You can rebase.

Reply:
Thanks!